client/my-sites/site-settings/form-writing.jsx

```diff
@@ -134,19 +135,21 @@ class SiteSettingsFormWriting extends Component {
 							isRequestingSettings={ isRequestingSettings }
 							fields={ fields }
 							jetpackVersionSupportsLazyImages={ jetpackVersionSupportsLazyImages }
+							siteAcceleratorSupported={ siteAcceleratorSupported }
 						/>
 					</div>
 				) }
 
 				{ jetpackSettingsUI &&
 					jetpackVersionSupportsLazyImages && (
 						<div>
-							{ this.renderSectionHeader( translate( 'Speed up your site' ), false ) }
+							{ this.renderSectionHeader( translate( 'Performance & speed' ), false ) }
 							<SpeedUpYourSite
 								isSavingSettings={ isSavingSettings }
 								isRequestingSettings={ isRequestingSettings }
 								fields={ fields }
 								jetpackVersionSupportsLazyImages={ jetpackVersionSupportsLazyImages }
+								siteAcceleratorSupported={ siteAcceleratorSupported }
```

Comment:
Seems like `<SpeedUpYourSite />` is the only component that cares about  `siteAcceleratorSupported` - how about retrieving it in its `connect()` instead, and removing it from `<SiteSettingsFormWriting />`?

Reply:
Yes, you're right. I fixed that.